src/controllers/feedController.js

```diff
@@ -0,0 +1,79 @@
+const { feedService } = require("../services");
+const { keyCheck } = require("../utils/keyCheck");
+
+const getFeedController = async(req, res) => {
+    const userId = req.params.userId;
+    const { page = 1 } = req.query;
+    const limit = 10
+    const offset = (page -1) * limit
+    console.log(page, offset);
+    const getFeed = await feedService.getFeedService(userId, limit, offset, page);
```

Comment:
```suggestion
    const feedData = await feedService.getFeedsByUserId(userId, limit, offset, page);
```
- 변수는 동사가 아닌 명사형으로 네이밍 하는 것이 일반적인 컨벤션입니다. 변수는 어떤 행동을 하지 않기 때문입니다.

Reply:
네 ! 변경하였습니다 !